an1.cc

```diff
@@ -832,18 +824,13 @@ static void Write_AN1_Waypoints(gbfile* f)
 
 static void Read_AN1_Lines(gbfile* f)
 {
-  unsigned long count = 0;
-  unsigned long i = 0;
-  unsigned long j = 0;
-  an1_line_record* rec = nullptr;
-  an1_vertex_record* vert = nullptr;
   route_head* rte_head;
   Waypoint* wpt_tmp;
 
   ReadShort(f);
-  count = ReadLong(f);
-  for (i = 0; i < count; i++) {
-    rec = Alloc_AN1_Line();
+  unsigned long count = ReadLong(f);
```

Comment:
Even if count wasn't used, was it needed to seek the read pointer ahead? (On a fone in bed. I may not have the whole picture. I doubt our coverage of an1 is very good.)

Reply:
It was the occurrence above in read waypoints that I meant to comment on.
Sorry.